integration_tests.py

```diff
@@ -528,9 +529,19 @@ def _gen_docker_compose(modules):
     # Write to a tempory file with the appropriate profile name
     prefix = "ci"
     with tempfile.TemporaryDirectory() as tmpdir:
+        input_docker_compose_dir = Path(__file__).parent / "tests/CI/"
         output_fn = Path(tmpdir) / prefix / "docker-compose.yml"
         output_fn.parent.mkdir()
         output_fn.write_text(yaml.safe_dump(docker_compose, sort_keys=False))
+        for env_file in input_docker_compose_dir.glob("*.env"):
```

Comment:
so you copy all the `env` files, until you find the good one and then stops ? Isn't it better to just look for the correct file from the beginning ?

Reply:
Yes, it would be better, but for whatever reason in https://github.com/DIRACGrid/DIRAC/pull/5308/files#diff-416917c73960f7e8baec3bb22e5c45e6bc5958f7ee87bbad81bd663846887953R22 I can't write
```
env_file: ${ES_VER}.env
```
so I ended up with this trick.

Reply:
Not even with quotes like https://docs.docker.com/compose/environment-variables/#substitute-environment-variables-in-compose-files  ?

Reply:
> Not even with quotes like https://docs.docker.com/compose/environment-variables/#substitute-environment-variables-in-compose-files ?

no, `env_file: "${ES_VER}".env` actually breaks it:

```
yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 17, column 5:
        image: ${ES_VER}
        ^
expected <block end>, but found '<scalar>'
  in "<unicode string>", line 22, column 26:
        env_file: "${ES_VER}".env
```

Looks like a bug in docker-compose to me.

> alternatively, if you do input_docker_compose_dir.glob("*%s*.env"%es_ver) you should have only one file,

I can only now `ES_VER` iff `docker_compose_env` is defined, which is not the case for `destroy`. Still, docker compose wants to find a file there even when destroying.

Reply:
`env_file: "${ES_VER}.env"`  that works with `docker-compose` at least

Reply:
OK thanks. I had to modify a few more things - in fact I was not paying enough attention and I mis-interpreted the error I was getting. Let's see now.